service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -161,6 +166,9 @@ public ResponseEntity<RunSetListResponse> getRunSets(UUID methodId, Integer page
 
   @Override
   public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
+    UserStatusInfo user = samService.getSamUser();
+    log.info("User ID: {}", user.getUserSubjectId());
```

Comment:
I would either remove this or add a `// TODO: remove/update in ticket WM-XXXX` comment, because I don't _think_ this is something we want there long term?